docs/dev_guide/migration/debug-with-codepin.rst

```diff
@@ -252,46 +251,109 @@ After building and executing ``dpct_output/example.dp.cpp`` and ``dpct_output_de
+                                            "Type": "int",
+                                            "Data": [
+                                                1
+                                            ]
+                                        }
+                                    },
+                                    {
+                                        "y": {
+                                            "Type": "int",
+                                            "Data": [
+                                                2
+                                            ]
+                                        }
+                                    },
           ...
 
      - .. code-block::
            :linenos:
 
-           {
-              "example.cu:23:3(SYCL):0": {
-                 "d_a[0]": {
-                    "m_Data": "01, 00, 00, 00, 02, 00, 00, 00, 03, 00, 00, 00"
-                 },
-                 "d_a[1]": {
-                    "m_Data": "01, 00, 00, 00, 02, 00, 00, 00, 03, 00, 00, 00"
-                 },
-                 "d_a[2]": {
-                    "m_Data": "01, 00, 00, 00, 02, 00, 00, 00, 03, 00, 00, 00"
-                 },
-                 "d_a[3]": {
-                    "m_Data": "00, 00, 00, 00, 00, 00, 00, 00, 00, 00, 00, 00"
-                 },
-                 "d_result[0]": {
-                    "m_Data": "00, 00, 00, 00, 00, 00, 00, 00, 00, 00, 00, 00"
-                 },
+        [
+            {
+                "ID": "example.cu:26:3:prolog",
+                "Free Device Memory": "0",
+                "Total Device Memory": "31023112192",
+                "Elapse Time(ms)": "0",
+                "CheckPoint": {
+                    "d_a": {
+                        "Type": "Pointer",
+                        "Data": [
+                            {
+                                "Type": "sycl::int3",
+                                "Data": [
+                                    {
+                                        "x": {
+                                            "Type": "int",
+                                            "Data": [
+                                                1
+                                            ]
+                                        }
+                                    },
+                                    {
+                                        "y": {
+                                            "Type": "int",
+                                            "Data": [
+                                                2
+                                            ]
+                                        }
+                                    },
             ...
 
 The report helps identify where the runtime behavior of the CUDA and the SYCL
-programs start to diverge from one another.
\ No newline at end of file
+programs start to diverge from one another.
+
+Analyze the Data Checkpoints
```

Comment:
L 322 should be updated as "Most Time-consuming Kernel" is not info of  Data Checkpoints . 
how about Analysis the  CodePin result?